content/tutorials/spatial_wms_services/index.md

```diff
@@ -98,8 +102,12 @@ leaflet() %>%
   )
 ```
 
-![Leaflet map with the orthophoto mosaic as
-background](index_files/figure-gfm/unnamed-chunk-5-1.png)
+<figure>
```

Comment:
Any reason for not keeping the original markdown code?

Reply:
See [discussion above](https://github.com/inbo/tutorials/pull/324#issue-1800513065); it is current knitting/pandoc behaviour. I have tried to control this but did not find a solution.

Reply:
Then @damianooldoni and @peterdesmet should review this.